### PR TITLE
feat(Text): support setting heading level to span and header pretitleAs

### PR DESCRIPTION
**Pull request by Winde**
(#1167)

### CODE REVIEW

File: src/community/advanced-data-card.tsx [hunk 1]
```diff
@@ -231,17 +231,15 @@ type AllowedExtra =
     | typeof SimpleBlock
     | typeof ValueBlock;
 
-type TextAs = 'h1' | 'h2' | 'h3' | 'h4';
```
Comment (Winde) [inline src/community/advanced-data-card.tsx:234]: This restriction of up to H4 was incoherent with the rest of mistica.

**Comment by Winde** [comment]
On hold for https://jira.tid.es/browse/PRODUCTDSN-2500

We expect  @MontseTelefonica  to reach out to @yceballost  to check what mistica documentation needs to be updated

**Review by vdedios — APPROVED** [review]

**Review by penchochris — APPROVED** [review]

**Review by pladaria — APPROVED** [review]

**Comment by yceballost** [comment]
> On hold for [jira.tid.es/browse/PRODUCTDSN-2500](https://jira.tid.es/browse/PRODUCTDSN-2500)
> 
> We expect @MontseTelefonica to reach out to @yceballost to check what mistica documentation needs to be updated

Yep, the specs should indicate that a title may not include hierarchy. The current specs are not updated with this definition.

@MontseTelefonica @aweell

**Review by atabel — APPROVED** [review]

**Pull request merged**